packages/block-library/src/archives/index.php

```diff
@@ -19,10 +19,6 @@ function render_block_core_archives( $attributes ) {
 
```

Comment:
A bit of a side note - I noticed that many of these blocks still have `align` defined in their attributes along with an "enum" array:
https://github.com/WordPress/gutenberg/blob/0d08cdab0c2aed8611ad4468b86e29247590c60b/packages/block-library/src/archives/block.json#L5-L14

I would have thought defining `align` in the attributes to no longer be necessary as the hook does this for us.  However, removing this will result in an `Invalid parameter(s): attributes` error for already existing blocks so I have not removed any of this.

However, that does make me wonder if this `enum` param is necessary in all cases and if it should be added to the align hook for defining the attribute when it does not exist? :

https://github.com/WordPress/gutenberg/blob/0d08cdab0c2aed8611ad4468b86e29247590c60b/packages/block-editor/src/hooks/align.js#L95-L97

Reply:
Could it be that blocks throwing errors just need to deprecate the old attributes set?
Something like this:
```jsx
const settings = {
  attributes: {
    foo: { type: 'string' },
  },
  deprecated: [
    {
      attributes: {
        align: { type: 'string', enum: [ /* ... */ ] },
        foo: { type: 'string' },
      },
    }
  ],
};
```

Reply:
Oh probably, Im more concerned if there was any other reason for keeping this definition (and not deprecating) when the `align` hook was originally added to these blocks.  And if that `enum` should be part of the hook itself.

Reply:
I feel the enum makes sense for all blocks. I think it adds some level of validation.

Reply:
and yes, removing the definition of these attributes also make sense.